src/fairseq2/recipes/lm/_online_finetune/_online_dpo.py

```diff
@@ -264,20 +264,26 @@ def __call__(
         )
         update_logit_entropy(metric_bag, tgt_logit_entropy)
 
+        prompt_lengths = (
+            prompt_batch.prompt_lengths
+            if is_bad_batch
+            else reward_output["prompt_lengths"]
+        )
```

Comment:
what's the difference between `prompt_batch.prompt_lengths` and `reward_output["prompt_lengths"]`?